### ISSUE TITLE
Main content navigation buttons "Left" and "Right" labels are not sufficiently clear

### ISSUE THREAD

**Issue by gautierchomel** [edrlab/thorium-reader#3108]
> The buttons to go to the next or previous page have labels that are "Left" and "Right", which is not sufficiently precise (especially given there are also the "jump forward" and "jump back" buttons. 
> 
> At first, the user could not determine the different functionality these would provide. This labelling ambiguity could be considered a violation of WCAG 1.3.1 (Information and Relationships) and 2.4.7 (Labels).
> 
> Suggestion: putting the main navigation buttons into a 'navigation' region or similar to help clarify their purpose in context.

Reported by ebsco.

**Comment by gautierchomel** [comment]
Alternatively, we could hide those buttons from assistive technologies as they don't provide a meaningful navigation experience (it's just next or previous displayed page).
